### DISCUSSION TITLE
Welcome to PcapXray Discussions!

### DISCUSSION THREAD

**Discussion by srixivas** [srixivas/PcapXray#58] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Problems/Bugs with installation or tool and tricks/solution that helped you solve it.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with any Q to get help.

**Comment by jalilel1913** [comment]
Just writing to say this is an AWESOME tool...
